test/common/stats/tag_extractor_impl_test.cc

```diff
@@ -346,6 +382,35 @@ TEST(TagExtractorTest, DefaultTagExtractors) {
 
   regex_tester.testRegex("listener_manager.worker_123.dispatcher.loop_duration_us",
                          "listener_manager.dispatcher.loop_duration_us", {worker_id});
+
+  // Listener worker id
+  listener_address.value_ = "127.0.0.1_33033";
+  worker_id.value_ = "worker_15";
+
+  regex_tester.testRegex("listener.127.0.0.1_33033.worker_15.downstream_cx_total",
+                         "listener.downstream_cx_total", {listener_address, worker_id});
+
+  // Server worker id
+  worker_id.value_ = "worker_3";
+
+  regex_tester.testRegex("server.worker_3.watchdog_mega_miss", "server.watchdog_mega_miss",
+                         {worker_id});
+
+  // Google GRPC Service
+  Tag google_grpc_service_prefix;
+  google_grpc_service_prefix.name_ = tag_names.GOOGLE_GRPC_SERVICE_PREFIX;
+  google_grpc_service_prefix.value_ = "sds_mesh_ca";
+
+  regex_tester.testRegex("grpc.sds_mesh_ca.streams_total", "grpc.streams_total",
+                         {google_grpc_service_prefix});
+
+  // GRPC status
+  Tag grpc_status;
+  grpc_status.name_ = tag_names.GRPC_STATUS;
+  grpc_status.value_ = "14";
+
+  regex_tester.testRegex("grpc.sds_mesh_ca.streams_closed_14", "grpc.streams_closed",
```

Comment:
can you confirm we have testcase covering the 11 new regexes you added

Reply:
Yes, I confirm that every new regex has a unit test